packages/cli/src/index.ts

```diff
@@ -567,6 +567,7 @@ const main = async () => {
           func = require('./commands/certs').default;
           break;
         case 'deploy':
+          telemetry.trackCliCommandDeploy(userSuppliedSubCommand);
```

Comment:
Not blocking but I don't think `deploy` has subcommands? Or does it count itself as a subcommand of `vercel` with no command?

Reply:
You're right, I wasn't really paying attention to what this is supposed to store. So I guess for anything that doesn't have a subcommand what value are we sending:

```
  trackCliCommandWhoami() {
    this.trackCliCommand({
      command: 'whoami',
      value: 'WHATSTHISVALUE',
    });
  }
  ```

Reply:
I think the value can just be the command again.